shared/charts/gpii-dataloader/templates/job.yaml

```diff
@@ -9,11 +9,33 @@ spec:
       labels:
         app: {{ template "dataloader.name" . }}
     spec:
+      shareProcessNamespace: true
       containers:
       - name: gpii-dataloader
         image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
         command: [ '/app/scripts/deleteAndLoadSnapsets.sh' ]
         env:
         - name: GPII_COUCHDB_URL
           value: '{{ .Values.couchdbUrl }}'
+      - name: istio-proxy-manager
+        image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 1337
+        command: [ "/bin/sh", "-c" ]
+        args:
+          - while ! ps -o user | grep node > /dev/null;
+            do
+              echo 'Waiting for deleteAndLoadSnapsets.sh to start...';
+              sleep 2;
+            done;
+            echo 'deleteAndLoadSnapsets.sh started.';
+            while ps -o user | grep node > /dev/null;
+            do
+              echo 'Waiting for deleteAndLoadSnapsets.sh to finish...';
+              sleep 2;
+            done;
+            echo 'deleteAndLoadSnapsets.sh finished.';
+            echo 'Sending TERM to pilot-agent';
+            pkill pilot-agent;
```

Comment:
Same question as here: https://github.com/gpii-ops/gpii-infra/pull/383#discussion_r309374031

Reply:
Same response:  added the loop.